diff --git a/easytier/Cargo.toml b/easytier/Cargo.toml
index a9a781d..0000000 100644
--- a/easytier/Cargo.toml
+++ b/easytier/Cargo.toml
@@ -202,7 +202,7 @@ sys-locale = "0.3"
 
 service-manager = { git = "https://github.com/EasyTier/service-manager-rs.git", branch = "main" }
 
-kcp-sys = { git = "https://github.com/EasyTier/kcp-sys", rev = "d7427c22d764deb1860a7d37acc446ed5033464c", optional = true }
+kcp-sys = { path = "../../kcp-sys", optional = true }
 
 # for dns connector
 hickory-resolver = { version = "0.25.2", optional = true }
